erts/emulator/beam/beam_emu.c

```diff
@@ -3516,7 +3535,8 @@ do {						\
 
  find_func_info: {
      SWAPOUT;
-     I = handle_error(c_p, I, reg, NULL);
+     I = handle_error(c_p, I, reg, mfa_bif);
```

Comment:
This change could be expensive. I would recommend copying the code, and inventing a new label (e.g. `arith_handle_error`).